IOS/SwiftSeedProject/SwiftSeedProject/Domain/Article.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Article.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 13/4/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class Article {
+    
+    var author: String?
```

Comment:
I'm not sure.. but you should be able to define all these fields as `let` right? Since you're only setting them inside the `init` method.

Also, can you define them as mandatory instead of optional?